packages/components/src/custom-select-control/index.js

```diff
@@ -257,3 +257,12 @@ export default function CustomSelectControl( props ) {
 		</div>
 	);
 }
+
+export default function CustomSelectControl( props ) {
+	return (
+		<ExperimentalCustomSelectControl
+			{ ...props }
+			showSelectedHint={ false }
+		/>
+	);
+}
```

Comment:
Could this be the other way around, where the default export is the experimental version named simply `CustomSelectControl`, and the named export is called `StableCustomSelectControl`?

In wp-components internally we will almost never use the non-experimental version, so I think it would cause less disruption if it were named the other way around. It would be error-prone if we had to remember to do `import { ExperimentalCustomSelectControl as CustomSelectControl }` every time some component had a locked version.

Reply:
Done. We'll just have to be careful when exporting a component that `{ ...props }` into an experimental component.